### PR TITLE
generalize template creation

### LINKED ISSUE #7: Add option to pull from local template copy instead of repo

### LINKED ISSUE #9: Generate new system/components through a template

### PR DESCRIPTION
**Pull request by tlein**
Resolves #9 
Resolves #7

### CODE REVIEW

File: Src/Raft/Path.ts [hunk 1]
```diff
@@ -111,5 +112,22 @@ class Path {
...
+    static home() : Path {
+        return new Path(os.homedir());
+    }
+
+    /**
+     * Gets the folder separator character for the given filesystem:
+     * / on *nix systems and \ on Windows.
+     * @return {string} File separator character.
+     */
+    static folderSeparator() : string {
```
Comment (jeffswenson) [inline Src/Raft/Path.ts:128]: It doesn't look like this is actually used anywhere.  You can delete it.

Comment (tlein) [inline Src/Raft/Path.ts:128]: The index.js file for the System template uses it:

https://github.com/tlein/AnconaTemplateGame/pull/11/files#diff-80b8ff8583212475cefc1fbc7a9fad68R27

**Comment by jeffswenson** [comment]
You should update the readme with instructions for how people can install the templates.

Tucker: Fixed

**Comment by jeffswenson** [comment]
When you use the system template the prompt includes an extra colon.

Tucker: unable to reproduce, it may be a platform dependent bug in Promptly

**Pull request merged**